### ISSUE TITLE
Noise profiles for Fujifilm X-E4

### ISSUE THREAD

**Issue by JacoTux** [darktable-org/darktable#12647]
For noise profiles, please also attach the entire archive with `.pdf` plots.
[darktable-noiseprofile-20221015.tar.gz](https://github.com/darktable-org/darktable/files/9793230/darktable-noiseprofile-20221015.tar.gz)

**Comment by rawfiner** [comment]
The profile looks good thanks!
Could you add the extended ISO values please (first value seems to be ISO 80 and last ISO 51200)? Thanks

**Comment by JacoTux** [comment]
Yes, but I have to do another series, this time complete.
1) because I am not at all satisfied with the print I made of the gradient necessary for the protocol, on a photo printer but on standard paper, the blacks on the left are not dense enough. I guess this is important.
2) because I have evacuated the extended ISO, mainly because of the 25600 and 51200, for the series to be homogeneous in exposure I need to find the right overexposure of its two boosted ISO.
But it will wait until the weekend, I'm only at the PRO on Wednesdays and Thursdays, when I could print the gradient in the best conditions.

Other recommendations for this new series
